test/specs/line.spec.js

```diff
@@ -587,6 +587,90 @@ define([
                     });
                 });
             });
+
+            describe('when axis labels aren\'t set', function() {
+
+                beforeEach(() => {
+                    dataset = aTestDataSet().withOneSource().build();
+                    lineChart = chart();
+                    // DOM Fixture Setup
+                    f = jasmine.getFixtures();
+                    f.fixturesPath = 'base/test/fixtures/';
+                    f.load('testContainer.html');
+
+                    containerFixture = d3.select('.test-container');
+                    containerFixture.datum(dataset).call(lineChart);
+                });
+
+                afterEach(() => {
+                    containerFixture.remove();
+                    f = jasmine.getFixtures();
+                    f.cleanUp();
+                    f.clearCache();
+                });
+
+                describe('on render', () => {
+
+                    it('should not render any axisLabel', () => {
+                        expect(containerFixture.selectAll('.axisLabel')["_groups"][0].length).toBe(0);
+                    });
+
+                    it('should provide axisLabelX getter and setter', () => {
```

Comment:
I guess this tests are not dependent on the labels being set or not, so we usually place them with the rest of API tests